x/leverage/simulation/operations.go

```diff
@@ -244,12 +249,16 @@ func SimulateMsgRepayAsset(ak simulation.AccountKeeper, bk types.BankKeeper, lk
 		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
 		accs []simtypes.Account, chainID string,
 	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
-		from, coin, skip := randomCollateralFields(r, ctx, accs, lk)
+		from, uToken, skip := randomCollateralFields(r, ctx, accs, lk)
```

Comment:
When simulating `MsgRepay`, the coin to repay should be a subset of `GetBorrowerBorrows(borrower.Address)`

I recommend creating a function `randomBorrowedFields` or something similar

Reply:
done at https://github.com/umee-network/umee/pull/433/commits/e0d03fa4126c475652d58d36906eeffe8ee33993